src/main/scala/cognite/spark/v1/StringDataPointsRelation.scala

```diff
@@ -24,6 +27,13 @@ case class StringDataPointsFilter(
     externalId: Option[String]
 )
 
+case class InsertStringDataPointsItem(
```

Comment:
Probably don't need this, results in the same schema as `StringDataPointsItem` and if we were reading back `java.sql.Timestamp` from the row we should probably continue to use `StringDataPointsItem` on line 53?